zulipterminal/core.py

```diff
@@ -173,6 +173,16 @@ def show_edit_history(
                             'Edit History (up/down scrolls)')
         )
 
+    def show_media_confirmation_popup(self, func: Any, command: str,
+                                      media_path: str) -> None:
+        callback = partial(func, self, command, media_path)
+        question = urwid.Text([
+            'Your requested media has been downloaded at ',
+            ('bold', media_path),
+            '. Do you want to view it?'
```

Comment:
Given the length of the download URLs for some files, this might well benefit from more vertical spacing, eg.
```
Your requested media has been downloaded to:
<filename>

Do you want the application to open it with <tool>?